lein-cloverage/src/leiningen/cloverage.clj

```diff
@@ -8,16 +8,24 @@
 (defn get-lib-version []
   (or (System/getenv "CLOVERAGE_VERSION") "RELEASE"))
 
+(defn already-has-cloverage? [project]
+  (seq (for [[id _version] (:dependencies project)
+             :when (= id 'cloverage/cloverage)]
+         true)))
+
 (defn ^:pass-through-help cloverage
   "Run code coverage on the project.
 
   To specify cloverage version, set the CLOVERAGE_VERSION environment variable.
```

Comment:
Should this docstring be changed? Previously, it was the case that the env var would override the project. That still makes sense to me, although for a long time a lot of people set CLOVERAGE_VERSION because there weren't any new releases and that was the way to get a SNAPSHOT (which had useful features like Coveralls support).

Specifically, it seems like the "right" priority list is env var, project, default current version. Right?

Reply:
Oh, yeah the docstring needs an update.

I don't think it's a good idea to set the version thru an environment variable, but since that functionality exists, this patch should probably prefer the env var version. However, I would strongly recommend removing the documentation for the env var support (I'll change the patch to do this if you agree) since it seems to be a workaround for the fact that the version requested in the project map is currently ignored.

Reply:
I tentatively agree, though I think we should actively deprecate, not simply hide the evidence. The reason it exists was because nobody was cutting releases for a while, that reason has gone away, and you should totally just go set it in your deps via `project.clj` now.